packages/cli/package.json

```diff
@@ -40,11 +40,15 @@
   ],
   "dependencies": {
     "@cosmjs/cosmwasm-launchpad": "^0.24.0-alpha.23",
+    "@cosmjs/cosmwasm-stargate": "^0.24.0-alpha.23",
     "@cosmjs/crypto": "^0.24.0-alpha.23",
     "@cosmjs/encoding": "^0.24.0-alpha.23",
     "@cosmjs/faucet-client": "^0.24.0-alpha.23",
     "@cosmjs/launchpad": "^0.24.0-alpha.23",
     "@cosmjs/math": "^0.24.0-alpha.23",
+    "@cosmjs/proto-signing": "^0.24.0-alpha.23",
+    "@cosmjs/stargate": "^0.24.0-alpha.23",
+    "@cosmjs/tendermint-rpc": "^0.24.0-alpha.23",
```

Comment:
`@cosmjs/tendermint-rpc` could be an optional dependency? Maybe it's unnecessarily complicated though.

Reply:
Thy optional? I added it here to allow users to access @cosmjs/tendermint-rpc functionality directly. And it is pollued in by `@cosmjs/stargate` anyways.

Reply:
I was thinking that since it's not directly depended on by any of the code here it's up to the user whether they want to be able to import it. But since it's installed anyway I guess there's no real benefit, it just makes things more complicated.

Reply:
The problem I see with packages that are not installed automatically is that they are very hard to install on demand, especially when using `npx` to start the CLI.